README.rst

```diff
@@ -497,25 +454,24 @@ If you choose to work with the main branch, we recommend using NVIDIA's PyTorch
     -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit \
     stack=67108864 --device=/dev/snd nvcr.io/nvidia/pytorch:23.10-py3
 
-Examples
---------
+Future Work
+-----------
 
-Many examples can be found under the `"Examples" <https://github.com/NVIDIA/NeMo/tree/stable/examples>`_ folder.
+The NeMo Framework Launcher does not currently support ASR and TTS training, but it will soon.
 
+Get Help
+---------
 
-Contributing
-------------
+FAQ can be found on the NeMo `Discussions board <https://github.com/NVIDIA/NeMo/discussions>`_. You are welcome to ask questions or start discussions on the board.
 
-We welcome community contributions! Please refer to `CONTRIBUTING.md <https://github.com/NVIDIA/NeMo/blob/stable/CONTRIBUTING.md>`_ for the process.
-
-Publications
```

Comment:
Add back publications, we have a new page for research publications

Reply:
OK